src/core/choroplethFeature.js

```diff
@@ -0,0 +1,247 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class choroplethFeature
+ *
+ * @class
+ * @extends geo.feature
+ * @returns {geo.choroplethFeature}
+ *
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo.choroplethFeature = function (arg) {
+  'use strict';
+  if (!(this instanceof geo.choroplethFeature)) {
+    return new geo.choroplethFeature(arg);
+  }
+  arg = arg || {};
+  geo.feature.call(this, arg);
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * @private
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  var m_this = this,
+      s_init = this._init,
+      s_data = this.data,
+      m_choropleth = $
+      .extend({},
+              {
+                minColor: 'black',
+                minOpacity: 0,
+                maxColor: 'white',
+                maxOpacity: 1,
+                /* 9-step based on paraview bwr colortable */
+                colorRange: [
+                  {r: 0.07514311, g: 0.468049805, b: 1},
+                  {r: 0.468487184, g: 0.588057293, b: 1},
+                  {r: 0.656658579, g: 0.707001303, b: 1},
+                  {r: 0.821573924, g: 0.837809045, b: 1},
+                  {r: 0.943467973, g: 0.943498599, b: 0.943398095},
+                  {r: 1, g: 0.788626485, b: 0.750707739},
+                  {r: 1, g: 0.6289553, b: 0.568237474},
+                  {r: 1, g: 0.472800903, b: 0.404551679},
+                  {r: 0.916482116, g: 0.236630659, b: 0.209939162}
+                ],
+                scale: d3.scale.quantize(),
+                value: function (geoJsonFeature) {
```

Comment:
What? That is just a dummy argument name for the function.  This particular function is the default accessor that returns the "value" of the polygon.

That said the default here assumes a geojson-like object structure, which differs from the other feature classes.  For consistency, I think it would be better to make the default something like this:

```javascript
function (d) {return d.value;};
```